sql-to-dbsp-compiler/SQL-compiler/src/main/codegen/includes/parserPostgresqlImpls.ftl

```diff
@@ -1,268 +0,0 @@
-<#--
-// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
why is this file no longer needed?

Reply:
Last time you asked why it was needed. We actually don't use any of these SQL constructs. You were right, it's not used. It comes from Calcite.